src/components/dialogs/network-modifications/static-var-compensator/creation/static-var-compensator-creation-dialog.tsx

```diff
@@ -0,0 +1,481 @@
+                });
+            });
+        },
+        [currentNodeUuid, editData, snackError, studyUuid]
+    );
+
+    const clear = useCallback(() => {
+        reset(emptyFormData);
+    }, [reset]);
+
+    const open = useOpenShortWaitFetching({
+        isDataFetched:
+            !isUpdate || editDataFetchStatus === FetchStatus.SUCCEED || editDataFetchStatus === FetchStatus.FAILED,
+        delay: FORM_LOADING_DELAY,
+    });
+
+    const onValidationError = useCallback(
+        (errors: any) => {
+            const tabsInError = [];
+            if (errors?.[CONNECTIVITY]) {
+                tabsInError.push(StaticVarCompensatorCreationDialogTab.CONNECTIVITY_TAB);
+            }
+            if (errors?.[SETPOINTS_LIMITS]) {
+                tabsInError.push(StaticVarCompensatorCreationDialogTab.SET_POINTS_LIMITS_TAB);
+            }
+            if (errors?.[AUTOMATON]) {
+                tabsInError.push(StaticVarCompensatorCreationDialogTab.AUTOMATON_TAB);
+            }
+            if (errors?.[ADDIONAL_INFOS]) {
+                tabsInError.push(StaticVarCompensatorCreationDialogTab.ADDITIONAL_INFO_TAB);
+            }
+
+            if (tabsInError.includes(tabIndex)) {
+                // error in current tab => do not change tab systematically but remove current tab in error list
+                setTabIndexesWithError(tabsInError.filter((errorTabIndex) => errorTabIndex !== tabIndex));
+            } else if (tabsInError.length > 0) {
+                // switch to the first tab in the list then remove the tab in the error list
+                setTabIndex(tabsInError[0]);
+                setTabIndexesWithError(tabsInError.filter((errorTabIndex, index, arr) => errorTabIndex !== arr[0]));
+            }
+        },
+        [tabIndex]
+    );
+
+    const headerAndTabs = (
+        <Grid container spacing={2}>
+            <StaticVarCompensatorCreationDialogHeader />
+            <StaticVarCompensatorCreationDialogTabs
+                tabIndex={tabIndex}
+                tabIndexesWithError={tabIndexesWithError}
+                setTabIndex={setTabIndex}
+            />
+        </Grid>
+    );
+
+    return (
+        <CustomFormProvider validationSchema={formSchema} {...formMethods}>
+            <ModificationDialog
+                fullWidth
+                maxWidth={'md'}
+                onClear={clear}
+                onSave={onSubmit}
+                aria-labelledby="dialog-create-staticCompensator"
+                titleId="CreateStaticVarCompensator"
+                subtitle={headerAndTabs}
+                searchCopy={searchCopy}
+                onValidationError={onValidationError}
+                open={open}
+                isDataFetching={isUpdate && editDataFetchStatus === FetchStatus.RUNNING}
+                PaperProps={{
+                    sx: {
+                        height: '55vh', // we want the dialog height to be fixed even when switching tabs
```

Comment:
You should remove the slider we get on a laptop screen for SetPointsLimitsForm and StandbyAutomatonForm

Reply:
May be set to be larger than 55vh in order to disable the scroller?

Reply:
I set the height to '95vh' like the other dialogs